custom_dc/load_data.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+# Script to process stats using simple stats loader.
+# Defaults
+MODE="customdc"
+OUTPUT_DIR=".data/output_$MODE"
+USAGE="Script to load data into custom dc using simple importer.
+Usage: $(basename $0) [Options]
+Options:
+  -c <file>       Json config file for stats importer
+  -e <file>       Load environment variables from file.
```

Comment:
Is there any default environment file to load such as:
custom_dc/cloudsql_env.list
custom_dc/sqlite_env.list

Reply:
@kmoscoe - Should we have a single `env.list` file that combines the 2 and update vars in there for sqlite and cloud sql? The docs will need to be updated accordingly.

Reply:
Yeah, I think it might make sense to just have a single file. Let's do that.

Reply:
Added env.list